File: Week 2/Label/Label - ObjC - HW/Label - ObjC - HW/ViewController.m
//
//  ViewController.m
//  Label - ObjC - HW
//
//  Created by Consultant on 6/6/22.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController


- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    float center = self.view.center.x - 50;
    CGRect frame = CGRectMake(center, 100, 200, 21);
    UILabel *label = [[UILabel alloc] initWithFrame:frame];
    [self applyLabelStyles:label];
    NSDate *today = [NSDate date];
    NSDateFormatter *dateFormat = [[NSDateFormatter alloc] init];
    [dateFormat setDateFormat:@"MM/dd/yyyy"];
    NSString *dateString = [dateFormat stringFromDate:today];
    label.text = dateString;
    [self.view addSubview:label];
    //1.3
    UILabel *label1, *label2;
    //1.4
    CGRect frame1 = CGRectMake(100, 130, 200, 21);
    label1 = [[UILabel alloc] initWithFrame:frame1];
    CGRect frame2 = CGRectMake(200, 130, 200, 21);
    label2 = [[UILabel alloc] initWithFrame:frame2];
    label1.text = @"Label 1";
    label2.text = @"Label 2";
    [self.view addSubview:label1];
    [self.view addSubview:label2];
    
    //1.5
    float forHeight = 160;
    for (int i = 0; i < 5; i++) {
        CGRect forFrame = CGRectMake(center, forHeight, 200, 21);
        UILabel *forLabel = [[UILabel alloc] initWithFrame:forFrame];
        [self applyLabelStyles:forLabel];
        forLabel.text = [NSString stringWithFormat:@"Label in for %d", i + 1];
        [self.view addSubview:forLabel];
        forHeight += 20.0;
    }

}

- (void) applyLabelStyles:(UILabel *) label{
    label.backgroundColor = UIColor.clearColor;
    label.textColor = UIColor.redColor;
    label.font = [UIFont fontWithName:@"Verdana" size:18.0];
    label.numberOfLines = 2;
    label.shadowColor = UIColor.lightGrayColor;
}


@end

File: Week 2/Label/Label - HW/Label - HW/ViewController.swift
//
//  ViewController.swift
//  Label - HW
//
//  Created by Consultant on 6/6/22.
//

import UIKit

class ViewController: UIViewController {

    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        let center = self.view.center.x - 50
        let frame = CGRect(x: center, y: 100, width: 200, height: 21)
        let label = UILabel(frame: frame)
        applyLabelStyles(label: label)
        let date = Date()
        let dateFormater = DateFormatter()
        dateFormater.dateFormat = "MM/dd/yyyy"
        label.text = dateFormater.string(from: date)
        view.addSubview(label)
        
        //1.3
        var label1: UILabel, label2: UILabel
        //1.4
        let frame1 = CGRect(x:100, y: 130, width: 300, height: 21)
        let frame2 = CGRect(x:200, y: 130, width: 300, height: 21)
        label1 = UILabel(frame: frame1)
        label2 = UILabel(frame: frame2)
        label1.text = "Label 1"
        label2.text = "Label 2"
        view.addSubview(label1)
        view.addSubview(label2)
        
        //1.5
        var forHeight = 160.0
        for i in 0..<5{
            let forFrame = CGRect(x:center, y: forHeight, width: 200, height: 21)
            let forLabel = UILabel(frame: forFrame)
            forLabel.text = "Label in for \(i + 1)"
            view.addSubview(forLabel)
            forHeight += 20.0
        }
        
        //1.6
        
        
        
    }
    
    func applyLabelStyles(label: UILabel){
        label.backgroundColor = .clear
        label.textColor = .red
        label.font = UIFont(name: "Verdana", size: 18)
        label.numberOfLines = 2
        label.shadowColor = .lightGray
    }
    



}

